assignments/this.js

```diff
@@ -1,26 +1,83 @@
 /* The for principles of "this";
 * in your own words. explain the four principle for the "this" keyword below.
 *
-* 1. 
-* 2. 
-* 3. 
-* 4. 
+* 1. Implicit Binding
+* 2. Explicit Binding
+* 3. new Binding
+* 4. window Binding
 *
 * write out a code example of each explanation above
 */
 
 // Principle 1
-
+//Window Binding: can use window.xyz to change the value you see
 // code example for Window Binding
-
-// Principle 2
-
-// code example for Implicit Binding
-
-// Principle 3
-
-// code example for New Binding
-
-// Principle 4
-
-// code example for Explicit Binding
\ No newline at end of file
+var sayRenter = function(){
+    console.log(this.lastRenter);
+  };
+  
+  var scoot315 = {
+    number: "scoot315",
+    lastRenter: "Tupac",
+    age: 45,
+    location: "37.7819° N, 122.4078° W",
+    startTime: "5:54am",
+    endTime: "6:13am"
+  }
+  sayRenter();
+  window.lastRenter = "Tupac doppleganger";
+  sayRenter();
+  
+  // Principle 2 
+  //Implicit Binding: anything to the left of the dot (.key) is being referenced.
+  // code example for Implicit Binding
+  var scoot231 = {
+    number: "scoot231",
+    lastRenter: "Sagar Desai",
+    age: 27,
+    location: "37.7623° N, 122.4577° W",
+    sayLocation: function(){
+      console.log(this.location)
+    },
+    startTime: "1:54pm",
+    endTime: "2:23pm"
+  }
+  scoot231.sayLocation();
```

Comment:
I can't fight you here. Would have prefered that sayLocation was outside of scoot231 but, you are technically correct and I love it. :+1: